ch04_Het_Beheer_en_Ontwikkelmodel.md

```diff
@@ -99,9 +99,9 @@ Onder de genoemde activiteiten verstaan we het volgende:
 
 De activiteiten moeten worden uitgevoerd door verschillende
 rollen. Een overzicht van rollen die relevant zijn bij het ontwikkelen
-en beheren van standaarden is opgenomen in NEN 7522:2021 "Medische
+en beheren van standaarden is opgenomen in [NEN 7522:2021 "Medische
 informatica - Ontwikkelen en beheren van standaarden en stelsels van
-standaarden", en is hier in licht aangepaste versie overgenomen.
+standaarden"](https://www.nen.nl/nen-7522-2021-nl-283706), en is hier in licht aangepaste versie overgenomen.
```

Comment:
Ok, maar waarom hier wel een markdown link en hiervoor niet?

Reply:
Omdat deze niet in een aside block staat en wel goed rendert